### ISSUE TITLE
Fails to 'Installing OS X binary package for R' on R:devel (exits with 1)

### ISSUE THREAD

**Issue by gvegayon** [travis-ci/travis-ci#6268]
Since the last couple of weeks I'm getting the following error:

```
Installing OS X binary package for R
0.00s$ sudo installer -pkg "/tmp/R.pkg" -target /
installer: Error the package path specified was invalid: '/tmp/R.pkg'.
The command "sudo installer -pkg "/tmp/R.pkg" -target /" failed and exited with 1 during .
```

The full log is here:
https://travis-ci.org/USCCANA/netdiffuseR/jobs/142572801#L44

All other platforms work fine (I build the package on R linux and OS using both versions of R, current and devel).

My .travis.yml follows:

https://github.com/USCCANA/netdiffuseR/blob/238663356dfa3c25934a802d1721de2a0ae0cd40/.travis.yml

Thanks

cc @craigcitro, @hadley and @jimhester

**Comment by solarce** [comment]
Could you try adding `osx_image: xcode7.3`? Using a more recent version of OSX and Xcode may help, our default is still 10.9/Xcode6.1

**Comment by gvegayon** [comment]
Keeps failing: https://travis-ci.org/USCCANA/netdiffuseR/jobs/144298597

**Comment by BanzaiMan** [comment]
https://cloud.r-project.org/bin/macosx/old/R-devel.pkg yields 404.

R maintainers will have to update the URL. Paging @craigcitro, @hadley @jimhester

**Comment by jimhester** [comment]
The automatic builds for R-devel seem to be broken at the moment (see http://r.research.att.com/, http://r.research.att.com/log-R-devel.mavericks.x86_64.html). This is presumably why they are not at the previous link as well.

I would suggest removing R-devel for OS X from your build matrix.

**Comment by solarce** [comment]
I am closing this given it is an upstream issue

**Comment by jimhester** [comment]
Additionally this should be fixed once #801 is merged and deployed.
